netbox/extras/forms/model_forms.py

```diff
@@ -299,18 +299,17 @@ class Meta:
     def init_script_choice(self):
```

Comment:
This should be updated to use an API-backed DynamicModelChoiceField like we do for webhooks.